### ISSUE TITLE
Crash while processing Redump's Sega - Mega CD & Sega CD - Discs (534) (2021-04-09 19-06-16)

### ISSUE THREAD

**Issue by Freeben666** [unexpectedpanda/retool#62]
**Describe the bug**
Retool v0.90 crashes unexpectedly when processing Redump's DAT for Sega - Mega CD & Sega CD - Discs (534) (2021-04-09 19-06-16)

**To reproduce**
Regions selected: France>Europe>World>Unknown
Languages selected: French + English
Exclusions: None
Modes: None
Custom global filters: Include "Ouendan"

**Expected behavior**
No crash...

**Operating system**
- Windows 10 20H2 19042.928

**Retool edition**
- retool-gui.exe v0.90

**Retool version**
Retool GUI: 0.09
Retool CLI: 0.90

**Are you using custom global or system filters? If so, list them below**

Global excludes:

Global includes: Ouendan

System excludes:

System includes:

**What other settings are you using?**
Regions selected: France>Europe>World>Unknown
Languages selected: French + English
Exclusions: None
Modes: None

**Additional context**
```
* Reading dat file: "D:\Emulation\Tools\Retool\DATs\Sega - Mega CD & Sega CD
  - Datfile (534) (2021-04-09 19-06-16).dat"
* Validating dat file... file is a Logiqx dat file.
* Converting dat file to a searchable format... done.
* Gathering stats... done.

|  DAT DETAILS
|  Description: Sega - Mega CD & Sega CD - Discs (534) (2021-04-09 19-06-16)
|  Author: redump.org
|  URL: http://redump.org/
|  Version: 2021-04-09 19-06-16

* Finding titles in regions... done.
* Finding parents across regions... done.
* Checking already excluded titles for includes... done.
Traceback (most recent call last):s... [17%]
  File "retool-gui.py", line 1002, in <module>
  File "retool-gui.py", line 610, in main
  File "retool.py", line 304, in main
  File "modules\parentselection.py", line 752, in assign_clones
ValueError: 'USA' is not in list
[7724] Failed to execute script retool-gui
```

**Comment by Freeben666** [comment]
Here's the exact DAT file used.
[Sega - Mega CD & Sega CD - Datfile (534) (2021-04-09 19-06-16).zip](https://github.com/unexpectedpanda/retool/files/6331959/Sega.-.Mega.CD.Sega.CD.-.Datfile.534.2021-04-09.19-06-16.zip)

**Comment by unexpectedpanda** [comment]
Ahh this is from the recent Mega CD/Sega CD changes. Shouldn't take long to fix.

**Comment by unexpectedpanda** [comment]
Fixed as of 0.91 :)
